vignettes/translators.Rmd

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
this flow is still pretty clunky -- ideally we'll get the interactive part of translate_package() modularized (e.g. as `po_add_translation(<language>)`) before release... that would probably entail a decent refactoring of this vignette. Do you want to merge this now anyway, or split off this vignette into a separate PR until we solve the interactive translation piece?

up to you, happy to merge as-is as of now.

Reply:
Let's merge now and continue to discuss the translator workflow. I'd also like to think about [poedit](https://poedit.net) because starting from machine translation seems pretty appealing, and I feel like we could figure out someway via RConsortium or RStudio to fund pro-licenses for those who needed it.